src/constants.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  RelationshipClass,
+  StepEntityMetadata,
+  StepRelationshipMetadata,
+} from '@jupiterone/integration-sdk-core';
+
+export const ACCOUNT_ENTITY_DATA_KEY = 'entity:account';
+export const COMPANY_ENTITY_DATA_KEY = 'entity:company';
+
+type EntityConstantKeys = 'ACCOUNT' | 'COMPANY' | 'USER' | 'FILE';
+
+export const entities: Record<EntityConstantKeys, StepEntityMetadata> = {
+  ACCOUNT: {
+    resourceName: 'Account',
+    _type: 'bamboohr_account',
+    _class: 'Account',
+  },
+  COMPANY: {
+    resourceName: 'Company',
+    _type: 'bamboohr_company',
+    _class: 'Organization',
+  },
+  USER: {
+    resourceName: 'User',
+    _type: 'bamboohr_user',
+    _class: 'User',
```

Comment:
Great call on going with the `User` entity here. In general, a JupiterOne account will have a single `Person` entity that maps out to many `Users`. The `Person` is created and managed internally, while integrations (almost) always have `User` entities.